yandextank/validator/validator.py

```diff
@@ -0,0 +1,159 @@
+import collections
+import importlib
+import os
+import pkgutil
+import sys
+import uuid
+
+import yaml
+from cerberus import Validator
+
+TANK_DIR = os.path.dirname(os.path.dirname(importlib.import_module('yandextank').__file__))
+
+class ValidationError(Exception):
+    pass
+
+
+def load_yaml_schema(directory, filename=None):
+    DEFAULT_FILENAME = 'schema.yaml'
+    name = filename if filename else DEFAULT_FILENAME
+    with open(os.path.join(directory, name), 'r') as f:
+        return yaml.load(f)
+
+
+def load_py(directory, filename=None):
+    DEFAULT_PY_MODULE_NAME = 'schema'
+    name = filename if filename else DEFAULT_PY_MODULE_NAME
+    path = os.path.join(directory, name)
+    schema_module = importlib.import_module(os.path.relpath(path, TANK_DIR).replace('/', '.'))
+    return schema_module.SCHEMA
+
+
+def load_schema(directory, filename=None):
+    try:
+        return load_yaml_schema(directory, filename)
+    except IOError:
+        try:
+            return load_py(directory, filename)
+        except ImportError:
+            raise IOError('Neither .yaml nor .py schema found in %s' % directory)
+
+
+class TankConfig(object):
+
+    def __init__(self, configs, with_dynamic_options=True):
+        if not isinstance(configs, list):
+            configs = [configs]
+        self.__raw_config_dict = self.__load_multiple(configs)
+        self.with_dynamic_options = with_dynamic_options
+        self._validated = None
+        self._plugins = None
+        self.BASE_SCHEMA = load_schema(os.path.join(os.path.dirname(__file__), '../core/'))
```

Comment:
I'm not sure this will work everywhere. You should use pkg_resource module. Your configs (yaml) will not be installed with PIP. Add them here:
https://github.com/yandex/yandex-tank/blob/develop/setup.py#L53-L65
And then use resource_filename, resource_string or resource_stream.
https://github.com/yandex/yandex-tank/blob/develop/yandextank/plugins/Aggregator/plugin.py#L63